File: gungnir/plugins/src/test/resources/sum.js
var total = 0;

function evaluate(value) {
  if (value != null && typeof value == 'number') {
    total += value;
  }
  return total;
}

function exclude(value) {
  if (value != null && typeof value == 'number') {
    total -= value;
  }
  return total;
}

function clear() {
  total = 0;
}

File: gungnir/plugins/src/test/resources/sum.py
total = 0

def evaluate(value):
  global total
  if value is not None and isinstance(value, (int, long, float, complex)):
    total += value
  return total

def exclude(value):
  global total
  if value is not None and isinstance(value, (int, long, float, complex)):
    total -= value
  return total

def clear():
  global total
  total = 0